source/main.cpp

```diff
@@ -168,6 +169,10 @@ int main(int argc, char *argv[])
 
 		Preferences::Load();
 
+		// Load global Conditions:
+		DataFile globCons(Files::Config() + "global conditions.txt");
+		GameData::GlobalConditions().Load(globCons, "global conditions");
```

Comment:
If we did it like this then we wouldn't need the new load function on ConditionsStore.
```suggestion
		DataFile globalConditions(Files::Config() + "global conditions.txt");
		for(const DataNode &node : globalConditions)
			if(node.Token(0) == "global conditions")
				GameData::GlobalConditions().Load(node);
```